src/containers/SessionMenu.js

```diff
@@ -68,6 +76,7 @@ SessionMenu.defaultProps = {
 function mapStateToProps(state) {
   return {
     isOpen: sessionMenuIsOpen(state),
+    currentNetwork: network(state),
```

Comment:
Why the preference for the selector here? They are generally for derived properties, but in this case we could just map directly to state: `state.network`'.